openedx/core/djangoapps/credit/api/provider.py

```diff
@@ -51,6 +50,35 @@ def get_credit_providers():
     return CreditProvider.get_credit_providers()
 
 
+def get_credit_provider_instructions(username, course_id):
```

Comment:
I think API should be get_credit_provider(provider_id) that will give all the detail of credit provider otherwise it will need other api to fetch the `provider_name`, `provider_url` and thumbnamil image etc